dp-cluster-setup-utility.py

```diff
@@ -1171,21 +1543,38 @@ def hdf_supported_version(self):
   def security_type_supported(self):
     return self.ambari.cluster.security_type == 'KERBEROS'
 
-  def running_on_knox_host(self):
-    if self.knox_host in (socket.gethostname(), socket.getfqdn()):
-      return True
-    if self.knox_ip() == socket.gethostbyname(socket.gethostname()):
-      return True
-    hostname, aliases, ips = socket.gethostbyname_ex(socket.gethostname())
-    if self.knox_host == hostname or self.knox_host in aliases or self.knox_ip() in ips:
-      return True
-    return False
 
-  def knox_ip(self):
-    try:
-      return socket.gethostbyname(self.knox_host)
-    except Exception:
-      return None
+class CMPrerequisites(BasePrerequisites):
+  def __init__(self, cm):
+    self.cm = cm
+    self.knox_host = cm.cluster.knox_host()
+
+  def satisfied(self):
+    if not self.stack_supported():
+      print('The stack version (%s) is not supported. Supported stacks are: CDH-5.17/CDH-6.3 or newer.' % self.cm.installed_stack())
+      return False
+    if not self.security_type_supported():
```

Comment:
Why have this check at all? Please remove. We can introduce when we need it for Knox integration.

Reply:
earlier we decided to put warning, will remove it